openfisca_france/model/prestations/minima_sociaux/ppa.py

```diff
@@ -198,6 +202,12 @@ class ppa_ressources_hors_activite_individu(Variable):
     entity = Individu
     label = u"Revenu hors activité pris en compte pour la PPA (Individu) pour un mois"
     definition_period = MONTH
+    reference = [
+        u"Article L842-4 du code de la sécurité sociale",
+        u"https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=B1D8827D50F7B3CC603BB7D398E71AA8.tplgfr28s_3?idArticle=LEGIARTI000033813782&cidTexte=LEGITEXT000006073189&dateTexte=20181226",
+        u"Article R843-1 du code de la sécurité sociale",
```

Comment:
```suggestion
        # Article R843-1 du code de la sécurité sociale"
```

Sinon, sur legislation explorer, on ajoute de liens morts :

<img width="808" alt="capture d ecran 2018-12-28 a 15 38 42" src="https://user-images.githubusercontent.com/329236/50518590-b21bc200-0ab6-11e9-99f4-2a3a41cdcd3f.png">